2014-09-02 09:11:37,912 [main] ERROR br.ufmg.hc.telessaude.diagnostico.dominio.exceptions.DAOException:28 - Unknown entity: br.ufmg.hc.telessaude.diagnostico.dominio.entity.HistoricoClinico
br.ufmg.hc.telessaude.diagnostico.dominio.exceptions.DAOException: Unknown entity: br.ufmg.hc.telessaude.diagnostico.dominio.entity.HistoricoClinico
	at br.ufmg.hc.telessaude.diagnostico.dominio.daoImpl.DaoBase.findById(DaoBase.java:232)
	at br.ufmg.hc.telessaude.diagnostico.dominio.daoImpl.HistoricoClinicoDAOImplTest.testConsultarPorIdExame(HistoricoClinicoDAOImplTest.java:26)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:45)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:42)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:263)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:68)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:47)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:231)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:60)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:50)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:222)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:300)
	at junit.framework.JUnit4TestAdapter.run(JUnit4TestAdapter.java:39)
	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.run(JUnitTestRunner.java:523)
	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.launch(JUnitTestRunner.java:1063)
	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.main(JUnitTestRunner.java:914)
2014-09-02 09:44:27,706 [main] ERROR br.ufmg.hc.telessaude.diagnostico.dominio.exceptions.DAOException:28 - could not execute query
br.ufmg.hc.telessaude.diagnostico.dominio.exceptions.DAOException: could not execute query
	at br.ufmg.hc.telessaude.diagnostico.dominio.daoImpl.ExameDAOImpl.findByRestrictions(ExameDAOImpl.java:188)
	at br.ufmg.hc.telessaude.diagnostico.dominio.daoImpl.DaoBase.findByRestrictions(DaoBase.java:145)
	at br.ufmg.hc.telessaude.diagnostico.dominio.daoImpl.ExameDAOImpl.consultarExamesConcluidos(ExameDAOImpl.java:131)
	at br.ufmg.hc.telessaude.diagnostico.dominio.daoImpl.ExameDAOImplTest.testConsultarExamesConcluidos(ExameDAOImplTest.java:59)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:45)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:42)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:28)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:30)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:263)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:68)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:47)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:231)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:60)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:50)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:222)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:28)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:30)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:300)
	at junit.framework.JUnit4TestAdapter.run(JUnit4TestAdapter.java:39)
	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.run(JUnitTestRunner.java:523)
	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.launch(JUnitTestRunner.java:1063)
	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.main(JUnitTestRunner.java:914)
2014-09-02 09:44:27,715 [main] ERROR br.ufmg.hc.telessaude.diagnostico.dominio.exceptions.DAOException:28 - could not execute query
br.ufmg.hc.telessaude.diagnostico.dominio.exceptions.DAOException: could not execute query
	at br.ufmg.hc.telessaude.diagnostico.dominio.daoImpl.ExameDAOImpl.findByRestrictions(ExameDAOImpl.java:188)
	at br.ufmg.hc.telessaude.diagnostico.dominio.daoImpl.ExameDAOImpl.consultarExamesNaoVisualizados(ExameDAOImpl.java:60)
	at br.ufmg.hc.telessaude.diagnostico.dominio.daoImpl.ExameDAOImpl.consultarExamesNaoVisualizados(ExameDAOImpl.java:47)
	at br.ufmg.hc.telessaude.diagnostico.dominio.daoImpl.ExameDAOImplTest.testConsultarExamesNaoVisualizados(ExameDAOImplTest.java:49)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:45)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:42)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:28)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:30)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:263)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:68)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:47)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:231)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:60)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:50)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:222)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:28)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:30)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:300)
	at junit.framework.JUnit4TestAdapter.run(JUnit4TestAdapter.java:39)
	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.run(JUnitTestRunner.java:523)
	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.launch(JUnitTestRunner.java:1063)
	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.main(JUnitTestRunner.java:914)
2014-09-02 09:45:21,376 [main] ERROR br.ufmg.hc.telessaude.diagnostico.dominio.exceptions.DAOException:28 - could not execute query
br.ufmg.hc.telessaude.diagnostico.dominio.exceptions.DAOException: could not execute query
	at br.ufmg.hc.telessaude.diagnostico.dominio.daoImpl.ExameDAOImpl.findByRestrictions(ExameDAOImpl.java:188)
	at br.ufmg.hc.telessaude.diagnostico.dominio.daoImpl.DaoBase.findByRestrictions(DaoBase.java:145)
	at br.ufmg.hc.telessaude.diagnostico.dominio.daoImpl.ExameDAOImpl.consultarExamesConcluidos(ExameDAOImpl.java:131)
	at br.ufmg.hc.telessaude.diagnostico.dominio.daoImpl.ExameDAOImplTest.testConsultarExamesConcluidos(ExameDAOImplTest.java:59)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:45)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:42)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:28)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:30)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:263)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:68)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:47)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:231)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:60)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:50)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:222)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:28)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:30)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:300)
	at junit.framework.JUnit4TestAdapter.run(JUnit4TestAdapter.java:39)
	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.run(JUnitTestRunner.java:523)
	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.launch(JUnitTestRunner.java:1063)
	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.main(JUnitTestRunner.java:914)
2014-09-02 09:45:21,422 [main] ERROR br.ufmg.hc.telessaude.diagnostico.dominio.exceptions.DAOException:28 - could not execute query
br.ufmg.hc.telessaude.diagnostico.dominio.exceptions.DAOException: could not execute query
	at br.ufmg.hc.telessaude.diagnostico.dominio.daoImpl.ExameDAOImpl.findByRestrictions(ExameDAOImpl.java:188)
	at br.ufmg.hc.telessaude.diagnostico.dominio.daoImpl.ExameDAOImpl.consultarExamesNaoVisualizados(ExameDAOImpl.java:60)
	at br.ufmg.hc.telessaude.diagnostico.dominio.daoImpl.ExameDAOImpl.consultarExamesNaoVisualizados(ExameDAOImpl.java:47)
	at br.ufmg.hc.telessaude.diagnostico.dominio.daoImpl.ExameDAOImplTest.testConsultarExamesNaoVisualizados(ExameDAOImplTest.java:49)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:45)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:42)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:28)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:30)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:263)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:68)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:47)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:231)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:60)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:50)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:222)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:28)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:30)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:300)
	at junit.framework.JUnit4TestAdapter.run(JUnit4TestAdapter.java:39)
	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.run(JUnitTestRunner.java:523)
	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.launch(JUnitTestRunner.java:1063)
	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.main(JUnitTestRunner.java:914)
2014-09-02 09:46:37,833 [main] ERROR br.ufmg.hc.telessaude.diagnostico.dominio.exceptions.DAOException:28 - could not execute query
br.ufmg.hc.telessaude.diagnostico.dominio.exceptions.DAOException: could not execute query
	at br.ufmg.hc.telessaude.diagnostico.dominio.daoImpl.ExameDAOImpl.findByRestrictions(ExameDAOImpl.java:188)
	at br.ufmg.hc.telessaude.diagnostico.dominio.daoImpl.DaoBase.findByRestrictions(DaoBase.java:145)
	at br.ufmg.hc.telessaude.diagnostico.dominio.daoImpl.ExameDAOImpl.consultarExamesConcluidos(ExameDAOImpl.java:131)
	at br.ufmg.hc.telessaude.diagnostico.dominio.daoImpl.ExameDAOImplTest.testConsultarExamesConcluidos(ExameDAOImplTest.java:59)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:45)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:42)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:28)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:30)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:263)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:68)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:47)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:231)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:60)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:50)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:222)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:28)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:30)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:300)
	at junit.framework.JUnit4TestAdapter.run(JUnit4TestAdapter.java:39)
	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.run(JUnitTestRunner.java:523)
	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.launch(JUnitTestRunner.java:1063)
	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.main(JUnitTestRunner.java:914)
2014-09-02 09:46:37,838 [main] ERROR br.ufmg.hc.telessaude.diagnostico.dominio.exceptions.DAOException:28 - could not execute query
br.ufmg.hc.telessaude.diagnostico.dominio.exceptions.DAOException: could not execute query
	at br.ufmg.hc.telessaude.diagnostico.dominio.daoImpl.ExameDAOImpl.findByRestrictions(ExameDAOImpl.java:188)
	at br.ufmg.hc.telessaude.diagnostico.dominio.daoImpl.ExameDAOImpl.consultarExamesNaoVisualizados(ExameDAOImpl.java:60)
	at br.ufmg.hc.telessaude.diagnostico.dominio.daoImpl.ExameDAOImpl.consultarExamesNaoVisualizados(ExameDAOImpl.java:47)
	at br.ufmg.hc.telessaude.diagnostico.dominio.daoImpl.ExameDAOImplTest.testConsultarExamesNaoVisualizados(ExameDAOImplTest.java:49)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:45)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:42)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:28)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:30)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:263)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:68)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:47)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:231)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:60)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:50)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:222)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:28)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:30)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:300)
	at junit.framework.JUnit4TestAdapter.run(JUnit4TestAdapter.java:39)
	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.run(JUnitTestRunner.java:523)
	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.launch(JUnitTestRunner.java:1063)
	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.main(JUnitTestRunner.java:914)
2014-09-02 09:46:37,872 [main] ERROR br.ufmg.hc.telessaude.diagnostico.dominio.exceptions.DAOException:28 - could not execute query
br.ufmg.hc.telessaude.diagnostico.dominio.exceptions.DAOException: could not execute query
	at br.ufmg.hc.telessaude.diagnostico.dominio.daoImpl.ExameDAOImpl.findByRestrictions(ExameDAOImpl.java:188)
	at br.ufmg.hc.telessaude.diagnostico.dominio.daoImpl.ExameDAOImpl.consultarHistoricoExames(ExameDAOImpl.java:116)
	at br.ufmg.hc.telessaude.diagnostico.dominio.daoImpl.ExameDAOImpl.consultarHistoricoExames(ExameDAOImpl.java:103)
	at br.ufmg.hc.telessaude.diagnostico.dominio.daoImpl.ExameDAOImplTest.testConsultarHistoricoExames(ExameDAOImplTest.java:54)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:45)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:42)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:28)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:30)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:263)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:68)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:47)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:231)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:60)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:50)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:222)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:28)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:30)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:300)
	at junit.framework.JUnit4TestAdapter.run(JUnit4TestAdapter.java:39)
	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.run(JUnitTestRunner.java:523)
	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.launch(JUnitTestRunner.java:1063)
	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.main(JUnitTestRunner.java:914)
2014-09-02 09:55:15,497 [main] ERROR br.ufmg.hc.telessaude.diagnostico.dominio.hibernate.HibernateUtil:42 - mappedBy reference an unknown target entity property: br.ufmg.hc.telessaude.diagnostico.dominio.entity.ConteudoExame."ID_EXAME" in br.ufmg.hc.telessaude.diagnostico.dominio.entity.Exame.conteudosExames
org.hibernate.AnnotationException: mappedBy reference an unknown target entity property: br.ufmg.hc.telessaude.diagnostico.dominio.entity.ConteudoExame."ID_EXAME" in br.ufmg.hc.telessaude.diagnostico.dominio.entity.Exame.conteudosExames
	at org.hibernate.cfg.annotations.CollectionBinder.bindStarToManySecondPass(CollectionBinder.java:578)
	at org.hibernate.cfg.annotations.CollectionBinder$1.secondPass(CollectionBinder.java:543)
	at org.hibernate.cfg.CollectionSecondPass.doSecondPass(CollectionSecondPass.java:66)
	at org.hibernate.cfg.Configuration.secondPassCompile(Configuration.java:1163)
	at org.hibernate.cfg.AnnotationConfiguration.secondPassCompile(AnnotationConfiguration.java:329)
	at org.hibernate.cfg.Configuration.buildSessionFactory(Configuration.java:1319)
	at org.hibernate.cfg.AnnotationConfiguration.buildSessionFactory(AnnotationConfiguration.java:867)
	at br.ufmg.hc.telessaude.diagnostico.dominio.hibernate.HibernateUtil.<clinit>(HibernateUtil.java:40)
	at br.ufmg.hc.telessaude.diagnostico.dominio.daoImpl.ExameDAOImpl.findByRestrictions(ExameDAOImpl.java:149)
	at br.ufmg.hc.telessaude.diagnostico.dominio.daoImpl.ExameDAOImpl.consultarHistoricoExames(ExameDAOImpl.java:116)
	at br.ufmg.hc.telessaude.diagnostico.dominio.daoImpl.ExameDAOImpl.consultarHistoricoExames(ExameDAOImpl.java:103)
	at br.ufmg.hc.telessaude.diagnostico.dominio.daoImpl.ExameDAOImplTest.testConsultarHistoricoExames(ExameDAOImplTest.java:54)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:45)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:42)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:28)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:30)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:263)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:68)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:47)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:231)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:60)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:50)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:222)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:28)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:30)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:300)
	at junit.framework.JUnit4TestAdapter.run(JUnit4TestAdapter.java:39)
	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.run(JUnitTestRunner.java:523)
	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.launch(JUnitTestRunner.java:1063)
	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.main(JUnitTestRunner.java:914)
2014-09-02 09:55:15,511 [main] ERROR br.ufmg.hc.telessaude.diagnostico.dominio.exceptions.DAOException:28 - 
br.ufmg.hc.telessaude.diagnostico.dominio.exceptions.DAOException
	at br.ufmg.hc.telessaude.diagnostico.dominio.daoImpl.DaoBase.findById(DaoBase.java:232)
	at br.ufmg.hc.telessaude.diagnostico.dominio.daoImpl.ExameDAOImplTest.testConsultarExameCompletoPorId(ExameDAOImplTest.java:76)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:45)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:42)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:28)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:30)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:263)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:68)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:47)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:231)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:60)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:50)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:222)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:28)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:30)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:300)
	at junit.framework.JUnit4TestAdapter.run(JUnit4TestAdapter.java:39)
	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.run(JUnitTestRunner.java:523)
	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.launch(JUnitTestRunner.java:1063)
	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.main(JUnitTestRunner.java:914)
2014-09-02 09:55:54,043 [main] ERROR br.ufmg.hc.telessaude.diagnostico.dominio.hibernate.HibernateUtil:42 - mappedBy reference an unknown target entity property: br.ufmg.hc.telessaude.diagnostico.dominio.entity.ConteudoExame."ID_CONTEUDOEXAME" in br.ufmg.hc.telessaude.diagnostico.dominio.entity.Exame.conteudosExames
org.hibernate.AnnotationException: mappedBy reference an unknown target entity property: br.ufmg.hc.telessaude.diagnostico.dominio.entity.ConteudoExame."ID_CONTEUDOEXAME" in br.ufmg.hc.telessaude.diagnostico.dominio.entity.Exame.conteudosExames
	at org.hibernate.cfg.annotations.CollectionBinder.bindStarToManySecondPass(CollectionBinder.java:578)
	at org.hibernate.cfg.annotations.CollectionBinder$1.secondPass(CollectionBinder.java:543)
	at org.hibernate.cfg.CollectionSecondPass.doSecondPass(CollectionSecondPass.java:66)
	at org.hibernate.cfg.Configuration.secondPassCompile(Configuration.java:1163)
	at org.hibernate.cfg.AnnotationConfiguration.secondPassCompile(AnnotationConfiguration.java:329)
	at org.hibernate.cfg.Configuration.buildSessionFactory(Configuration.java:1319)
	at org.hibernate.cfg.AnnotationConfiguration.buildSessionFactory(AnnotationConfiguration.java:867)
	at br.ufmg.hc.telessaude.diagnostico.dominio.hibernate.HibernateUtil.<clinit>(HibernateUtil.java:40)
	at br.ufmg.hc.telessaude.diagnostico.dominio.daoImpl.ExameDAOImpl.findByRestrictions(ExameDAOImpl.java:149)
	at br.ufmg.hc.telessaude.diagnostico.dominio.daoImpl.ExameDAOImpl.consultarHistoricoExames(ExameDAOImpl.java:116)
	at br.ufmg.hc.telessaude.diagnostico.dominio.daoImpl.ExameDAOImpl.consultarHistoricoExames(ExameDAOImpl.java:103)
	at br.ufmg.hc.telessaude.diagnostico.dominio.daoImpl.ExameDAOImplTest.testConsultarHistoricoExames(ExameDAOImplTest.java:54)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:45)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:42)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:28)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:30)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:263)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:68)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:47)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:231)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:60)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:50)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:222)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:28)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:30)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:300)
	at junit.framework.JUnit4TestAdapter.run(JUnit4TestAdapter.java:39)
	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.run(JUnitTestRunner.java:523)
	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.launch(JUnitTestRunner.java:1063)
	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.main(JUnitTestRunner.java:914)
2014-09-02 09:55:54,056 [main] ERROR br.ufmg.hc.telessaude.diagnostico.dominio.exceptions.DAOException:28 - 
br.ufmg.hc.telessaude.diagnostico.dominio.exceptions.DAOException
	at br.ufmg.hc.telessaude.diagnostico.dominio.daoImpl.DaoBase.findById(DaoBase.java:232)
	at br.ufmg.hc.telessaude.diagnostico.dominio.daoImpl.ExameDAOImplTest.testConsultarExameCompletoPorId(ExameDAOImplTest.java:76)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:45)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:42)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:28)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:30)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:263)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:68)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:47)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:231)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:60)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:50)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:222)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:28)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:30)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:300)
	at junit.framework.JUnit4TestAdapter.run(JUnit4TestAdapter.java:39)
	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.run(JUnitTestRunner.java:523)
	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.launch(JUnitTestRunner.java:1063)
	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.main(JUnitTestRunner.java:914)
2014-09-19 12:24:32,964 [main] ERROR br.ufmg.hc.telessaude.diagnostico.dominio.exceptions.DAOException:28 - could not execute query
br.ufmg.hc.telessaude.diagnostico.dominio.exceptions.DAOException: could not execute query
	at br.ufmg.hc.telessaude.diagnostico.dominio.daoImpl.DaoBase.findAll(DaoBase.java:113)
	at br.ufmg.hc.telessaude.teletransferencia.daoImpl.ConfiguracaoTransferenciaDAOImplTest.testConsultarTodos(ConfiguracaoTransferenciaDAOImplTest.java:69)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:45)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:42)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:263)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:68)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:47)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:231)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:60)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:50)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:222)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:300)
	at junit.framework.JUnit4TestAdapter.run(JUnit4TestAdapter.java:39)
	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.run(JUnitTestRunner.java:523)
	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.launch(JUnitTestRunner.java:1063)
	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.main(JUnitTestRunner.java:914)
2014-09-19 12:24:32,985 [main] ERROR br.ufmg.hc.telessaude.diagnostico.dominio.exceptions.DAOException:28 - could not execute query
br.ufmg.hc.telessaude.diagnostico.dominio.exceptions.DAOException: could not execute query
	at br.ufmg.hc.telessaude.diagnostico.dominio.daoImpl.DaoBase.findByRestrictions(DaoBase.java:216)
	at br.ufmg.hc.telessaude.diagnostico.dominio.daoImpl.DaoBase.findByRestrictions(DaoBase.java:145)
	at br.ufmg.hc.telessaude.teletransferencia.daoImpl.ConfiguracaoTransferenciaDAOImpl.consultarPelaChave(ConfiguracaoTransferenciaDAOImpl.java:34)
	at br.ufmg.hc.telessaude.teletransferencia.daoImpl.ConfiguracaoTransferenciaDAOImplTest.testConsultarPelaChave(ConfiguracaoTransferenciaDAOImplTest.java:33)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:45)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:42)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:263)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:68)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:47)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:231)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:60)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:50)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:222)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:300)
	at junit.framework.JUnit4TestAdapter.run(JUnit4TestAdapter.java:39)
	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.run(JUnitTestRunner.java:523)
	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.launch(JUnitTestRunner.java:1063)
	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.main(JUnitTestRunner.java:914)
2014-09-19 12:24:33,078 [main] ERROR br.ufmg.hc.telessaude.diagnostico.dominio.exceptions.DAOException:28 - could not execute query
br.ufmg.hc.telessaude.diagnostico.dominio.exceptions.DAOException: could not execute query
	at br.ufmg.hc.telessaude.teletransferencia.daoImpl.ConfiguracaoTransferenciaDAOImpl.consultarQuantidade(ConfiguracaoTransferenciaDAOImpl.java:60)
	at br.ufmg.hc.telessaude.teletransferencia.daoImpl.ConfiguracaoTransferenciaDAOImplTest.testConsultarQuantidade(ConfiguracaoTransferenciaDAOImplTest.java:59)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:45)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:42)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:263)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:68)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:47)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:231)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:60)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:50)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:222)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:300)
	at junit.framework.JUnit4TestAdapter.run(JUnit4TestAdapter.java:39)
	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.run(JUnitTestRunner.java:523)
	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.launch(JUnitTestRunner.java:1063)
	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.main(JUnitTestRunner.java:914)
2014-11-04 08:23:47,644 [main] ERROR br.ufmg.hc.telessaude.diagnostico.dominio.exceptions.DAOException:28 - 
br.ufmg.hc.telessaude.diagnostico.dominio.exceptions.DAOException
	at br.ufmg.hc.telessaude.diagnostico.dominio.daoImpl.ExameDAOImpl.salvarExame(ExameDAOImpl.java:346)
	at br.ufmg.hc.telessaude.diagnostico.dominio.daoImpl.ExameDAOImplTest.testSalvarExame(ExameDAOImplTest.java:271)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:45)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:42)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:28)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:30)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:263)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:68)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:47)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:231)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:60)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:50)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:222)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:28)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:30)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:300)
	at junit.framework.JUnit4TestAdapter.run(JUnit4TestAdapter.java:39)
	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.run(JUnitTestRunner.java:523)
	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.launch(JUnitTestRunner.java:1063)
	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.main(JUnitTestRunner.java:914)
2014-11-04 11:52:41,959 [main] ERROR br.ufmg.hc.telessaude.diagnostico.dominio.exceptions.DAOException:28 - not-null property references a null or transient value: br.ufmg.hc.telessaude.diagnostico.dominio.entity.Paciente.datanascimento
br.ufmg.hc.telessaude.diagnostico.dominio.exceptions.DAOException: not-null property references a null or transient value: br.ufmg.hc.telessaude.diagnostico.dominio.entity.Paciente.datanascimento
	at br.ufmg.hc.telessaude.diagnostico.dominio.daoImpl.ExameDAOImpl.salvarExame(ExameDAOImpl.java:348)
	at br.ufmg.hc.telessaude.diagnostico.dominio.daoImpl.ExameDAOImplTest.testSalvarExame(ExameDAOImplTest.java:275)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:45)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:42)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:28)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:30)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:263)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:68)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:47)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:231)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:60)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:50)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:222)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:28)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:30)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:300)
	at junit.framework.JUnit4TestAdapter.run(JUnit4TestAdapter.java:39)
	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.run(JUnitTestRunner.java:523)
	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.launch(JUnitTestRunner.java:1063)
	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.main(JUnitTestRunner.java:914)
2014-11-04 11:54:41,476 [main] ERROR br.ufmg.hc.telessaude.diagnostico.dominio.exceptions.DAOException:28 - Could not execute JDBC batch update
br.ufmg.hc.telessaude.diagnostico.dominio.exceptions.DAOException: Could not execute JDBC batch update
	at br.ufmg.hc.telessaude.diagnostico.dominio.daoImpl.ExameDAOImpl.salvarExame(ExameDAOImpl.java:348)
	at br.ufmg.hc.telessaude.diagnostico.dominio.daoImpl.ExameDAOImplTest.testSalvarExame(ExameDAOImplTest.java:278)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:45)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:42)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:28)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:30)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:263)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:68)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:47)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:231)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:60)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:50)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:222)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:28)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:30)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:300)
	at junit.framework.JUnit4TestAdapter.run(JUnit4TestAdapter.java:39)
	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.run(JUnitTestRunner.java:523)
	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.launch(JUnitTestRunner.java:1063)
	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.main(JUnitTestRunner.java:914)
2014-11-04 11:56:18,840 [main] ERROR br.ufmg.hc.telessaude.diagnostico.dominio.exceptions.DAOException:28 - not-null property references a null or transient value: br.ufmg.hc.telessaude.diagnostico.dominio.entity.Exame.pontoRemoto
br.ufmg.hc.telessaude.diagnostico.dominio.exceptions.DAOException: not-null property references a null or transient value: br.ufmg.hc.telessaude.diagnostico.dominio.entity.Exame.pontoRemoto
	at br.ufmg.hc.telessaude.diagnostico.dominio.daoImpl.ExameDAOImpl.salvarExame(ExameDAOImpl.java:348)
	at br.ufmg.hc.telessaude.diagnostico.dominio.daoImpl.ExameDAOImplTest.testSalvarExame(ExameDAOImplTest.java:280)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:45)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:42)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:28)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:30)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:263)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:68)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:47)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:231)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:60)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:50)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:222)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:28)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:30)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:300)
	at junit.framework.JUnit4TestAdapter.run(JUnit4TestAdapter.java:39)
	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.run(JUnitTestRunner.java:523)
	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.launch(JUnitTestRunner.java:1063)
	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.main(JUnitTestRunner.java:914)
2014-11-04 11:57:40,656 [main] ERROR br.ufmg.hc.telessaude.diagnostico.dominio.exceptions.DAOException:28 - not-null property references a null or transient value: br.ufmg.hc.telessaude.diagnostico.dominio.entity.Exame.tipoExame
br.ufmg.hc.telessaude.diagnostico.dominio.exceptions.DAOException: not-null property references a null or transient value: br.ufmg.hc.telessaude.diagnostico.dominio.entity.Exame.tipoExame
	at br.ufmg.hc.telessaude.diagnostico.dominio.daoImpl.ExameDAOImpl.salvarExame(ExameDAOImpl.java:348)
	at br.ufmg.hc.telessaude.diagnostico.dominio.daoImpl.ExameDAOImplTest.testSalvarExame(ExameDAOImplTest.java:285)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:45)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:42)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:28)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:30)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:263)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:68)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:47)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:231)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:60)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:50)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:222)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:28)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:30)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:300)
	at junit.framework.JUnit4TestAdapter.run(JUnit4TestAdapter.java:39)
	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.run(JUnitTestRunner.java:523)
	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.launch(JUnitTestRunner.java:1063)
	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.main(JUnitTestRunner.java:914)
2014-11-04 12:01:01,651 [main] ERROR br.ufmg.hc.telessaude.diagnostico.dominio.exceptions.DAOException:28 - 
br.ufmg.hc.telessaude.diagnostico.dominio.exceptions.DAOException
	at br.ufmg.hc.telessaude.diagnostico.dominio.daoImpl.ExameDAOImpl.salvarExame(ExameDAOImpl.java:348)
	at br.ufmg.hc.telessaude.diagnostico.dominio.daoImpl.ExameDAOImplTest.testSalvarExame(ExameDAOImplTest.java:287)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:45)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:42)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:28)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:30)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:263)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:68)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:47)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:231)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:60)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:50)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:222)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:28)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:30)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:300)
	at junit.framework.JUnit4TestAdapter.run(JUnit4TestAdapter.java:39)
	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.run(JUnitTestRunner.java:523)
	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.launch(JUnitTestRunner.java:1063)
	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.main(JUnitTestRunner.java:914)
2014-11-04 12:24:06,968 [main] ERROR br.ufmg.hc.telessaude.diagnostico.dominio.exceptions.DAOException:28 - 
br.ufmg.hc.telessaude.diagnostico.dominio.exceptions.DAOException
	at br.ufmg.hc.telessaude.diagnostico.dominio.daoImpl.ExameDAOImpl.salvarExame(ExameDAOImpl.java:356)
	at br.ufmg.hc.telessaude.diagnostico.dominio.daoImpl.ExameDAOImplTest.testSalvarExame(ExameDAOImplTest.java:285)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:45)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:42)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:28)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:30)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:263)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:68)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:47)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:231)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:60)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:50)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:222)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:28)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:30)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:300)
	at junit.framework.JUnit4TestAdapter.run(JUnit4TestAdapter.java:39)
	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.run(JUnitTestRunner.java:523)
	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.launch(JUnitTestRunner.java:1063)
	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.main(JUnitTestRunner.java:914)
2014-11-04 12:26:05,352 [main] ERROR br.ufmg.hc.telessaude.diagnostico.dominio.exceptions.DAOException:28 - Could not execute JDBC batch update
br.ufmg.hc.telessaude.diagnostico.dominio.exceptions.DAOException: Could not execute JDBC batch update
	at br.ufmg.hc.telessaude.diagnostico.dominio.daoImpl.ExameDAOImpl.salvarExame(ExameDAOImpl.java:356)
	at br.ufmg.hc.telessaude.diagnostico.dominio.daoImpl.ExameDAOImplTest.testSalvarExame(ExameDAOImplTest.java:290)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:45)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:42)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:28)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:30)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:263)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:68)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:47)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:231)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:60)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:50)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:222)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:28)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:30)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:300)
	at junit.framework.JUnit4TestAdapter.run(JUnit4TestAdapter.java:39)
	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.run(JUnitTestRunner.java:523)
	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.launch(JUnitTestRunner.java:1063)
	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.main(JUnitTestRunner.java:914)
2014-11-04 12:26:51,107 [main] ERROR br.ufmg.hc.telessaude.diagnostico.dominio.exceptions.DAOException:28 - Could not execute JDBC batch update
br.ufmg.hc.telessaude.diagnostico.dominio.exceptions.DAOException: Could not execute JDBC batch update
	at br.ufmg.hc.telessaude.diagnostico.dominio.daoImpl.ExameDAOImpl.salvarExame(ExameDAOImpl.java:356)
	at br.ufmg.hc.telessaude.diagnostico.dominio.daoImpl.ExameDAOImplTest.testSalvarExame(ExameDAOImplTest.java:290)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:45)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:42)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:28)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:30)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:263)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:68)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:47)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:231)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:60)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:50)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:222)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:28)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:30)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:300)
	at junit.framework.JUnit4TestAdapter.run(JUnit4TestAdapter.java:39)
	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.run(JUnitTestRunner.java:523)
	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.launch(JUnitTestRunner.java:1063)
	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.main(JUnitTestRunner.java:914)
2014-11-04 12:28:01,666 [main] ERROR br.ufmg.hc.telessaude.diagnostico.dominio.exceptions.DAOException:28 - Could not execute JDBC batch update
br.ufmg.hc.telessaude.diagnostico.dominio.exceptions.DAOException: Could not execute JDBC batch update
	at br.ufmg.hc.telessaude.diagnostico.dominio.daoImpl.ExameDAOImpl.salvarExame(ExameDAOImpl.java:356)
	at br.ufmg.hc.telessaude.diagnostico.dominio.daoImpl.ExameDAOImplTest.testSalvarExame(ExameDAOImplTest.java:290)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:45)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:42)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:28)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:30)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:263)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:68)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:47)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:231)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:60)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:50)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:222)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:28)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:30)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:300)
	at junit.framework.JUnit4TestAdapter.run(JUnit4TestAdapter.java:39)
	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.run(JUnitTestRunner.java:523)
	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.launch(JUnitTestRunner.java:1063)
	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.main(JUnitTestRunner.java:914)
2014-11-04 12:31:08,568 [main] ERROR br.ufmg.hc.telessaude.diagnostico.dominio.exceptions.DAOException:28 - Could not execute JDBC batch update
br.ufmg.hc.telessaude.diagnostico.dominio.exceptions.DAOException: Could not execute JDBC batch update
	at br.ufmg.hc.telessaude.diagnostico.dominio.daoImpl.ExameDAOImpl.salvarExame(ExameDAOImpl.java:356)
	at br.ufmg.hc.telessaude.diagnostico.dominio.daoImpl.ExameDAOImplTest.testSalvarExame(ExameDAOImplTest.java:290)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:45)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:42)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:28)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:30)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:263)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:68)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:47)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:231)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:60)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:50)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:222)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:28)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:30)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:300)
	at junit.framework.JUnit4TestAdapter.run(JUnit4TestAdapter.java:39)
	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.run(JUnitTestRunner.java:523)
	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.launch(JUnitTestRunner.java:1063)
	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.main(JUnitTestRunner.java:914)
2015-05-01 22:39:58,368 [main] ERROR conversorbdweka.hibernate.HibernateUtil:42 - Unable to find physical table: `ESPECIALISTA`
org.hibernate.MappingException: Unable to find physical table: `ESPECIALISTA`
	at org.hibernate.cfg.Configuration$MappingsImpl.getLogicalTableName(Configuration.java:3430)
	at org.hibernate.cfg.Configuration$MappingsImpl.getLogicalTableName(Configuration.java:3423)
	at org.hibernate.cfg.annotations.CollectionBinder.bindManyToManySecondPass(CollectionBinder.java:1258)
	at org.hibernate.cfg.annotations.CollectionBinder.bindStarToManySecondPass(CollectionBinder.java:710)
	at org.hibernate.cfg.annotations.CollectionBinder$1.secondPass(CollectionBinder.java:645)
	at org.hibernate.cfg.CollectionSecondPass.doSecondPass(CollectionSecondPass.java:65)
	at org.hibernate.cfg.Configuration.originalSecondPassCompile(Configuration.java:1716)
	at org.hibernate.cfg.Configuration.secondPassCompile(Configuration.java:1423)
	at org.hibernate.cfg.Configuration.buildSessionFactory(Configuration.java:1856)
	at conversorbdweka.hibernate.HibernateUtil.<clinit>(HibernateUtil.java:40)
	at conversorbdweka.dominio.origem.daoimpl.DaoBase.findAll(DaoBase.java:112)
	at conversorbdweka.ConversorBdWeka.main(ConversorBdWeka.java:27)
2015-05-01 22:39:58,375 [main] ERROR conversorbdweka.utilidades.DAOException:27 - 
conversorbdweka.utilidades.DAOException
	at conversorbdweka.dominio.origem.daoimpl.DaoBase.findAll(DaoBase.java:114)
	at conversorbdweka.ConversorBdWeka.main(ConversorBdWeka.java:27)
2015-05-01 22:48:03,410 [main] ERROR conversorbdweka.hibernate.HibernateUtil:42 - Unable to find physical table: `ESPECIALISTA`
org.hibernate.MappingException: Unable to find physical table: `ESPECIALISTA`
	at org.hibernate.cfg.Configuration$MappingsImpl.getLogicalTableName(Configuration.java:3430)
	at org.hibernate.cfg.Configuration$MappingsImpl.getLogicalTableName(Configuration.java:3423)
	at org.hibernate.cfg.annotations.CollectionBinder.bindManyToManySecondPass(CollectionBinder.java:1258)
	at org.hibernate.cfg.annotations.CollectionBinder.bindStarToManySecondPass(CollectionBinder.java:710)
	at org.hibernate.cfg.annotations.CollectionBinder$1.secondPass(CollectionBinder.java:645)
	at org.hibernate.cfg.CollectionSecondPass.doSecondPass(CollectionSecondPass.java:65)
	at org.hibernate.cfg.Configuration.originalSecondPassCompile(Configuration.java:1716)
	at org.hibernate.cfg.Configuration.secondPassCompile(Configuration.java:1423)
	at org.hibernate.cfg.Configuration.buildSessionFactory(Configuration.java:1856)
	at conversorbdweka.hibernate.HibernateUtil.<clinit>(HibernateUtil.java:40)
	at conversorbdweka.dominio.origem.daoimpl.DaoBase.findAll(DaoBase.java:112)
	at conversorbdweka.ConversorBdWeka.main(ConversorBdWeka.java:27)
2015-05-01 22:49:59,212 [main] ERROR conversorbdweka.hibernate.HibernateUtil:42 - Unable to find physical table: `ESPECIALISTA`
org.hibernate.MappingException: Unable to find physical table: `ESPECIALISTA`
	at org.hibernate.cfg.Configuration$MappingsImpl.getLogicalTableName(Configuration.java:3430)
	at org.hibernate.cfg.Configuration$MappingsImpl.getLogicalTableName(Configuration.java:3423)
	at org.hibernate.cfg.annotations.CollectionBinder.bindManyToManySecondPass(CollectionBinder.java:1258)
	at org.hibernate.cfg.annotations.CollectionBinder.bindStarToManySecondPass(CollectionBinder.java:710)
	at org.hibernate.cfg.annotations.CollectionBinder$1.secondPass(CollectionBinder.java:645)
	at org.hibernate.cfg.CollectionSecondPass.doSecondPass(CollectionSecondPass.java:65)
	at org.hibernate.cfg.Configuration.originalSecondPassCompile(Configuration.java:1716)
	at org.hibernate.cfg.Configuration.secondPassCompile(Configuration.java:1423)
	at org.hibernate.cfg.Configuration.buildSessionFactory(Configuration.java:1856)
	at conversorbdweka.hibernate.HibernateUtil.<clinit>(HibernateUtil.java:40)
	at conversorbdweka.dominio.origem.daoimpl.DaoBase.findAll(DaoBase.java:112)
	at conversorbdweka.ConversorBdWeka.main(ConversorBdWeka.java:27)
2015-05-01 22:49:59,216 [main] ERROR conversorbdweka.utilidades.DAOException:27 - 
conversorbdweka.utilidades.DAOException
	at conversorbdweka.dominio.origem.daoimpl.DaoBase.findAll(DaoBase.java:114)
	at conversorbdweka.ConversorBdWeka.main(ConversorBdWeka.java:27)
2015-05-01 22:52:15,764 [main] ERROR conversorbdweka.hibernate.HibernateUtil:42 - Unable to find physical table: `ESPECIALISTA`
org.hibernate.MappingException: Unable to find physical table: `ESPECIALISTA`
	at org.hibernate.cfg.Configuration$MappingsImpl.getLogicalTableName(Configuration.java:3430)
	at org.hibernate.cfg.Configuration$MappingsImpl.getLogicalTableName(Configuration.java:3423)
	at org.hibernate.cfg.annotations.CollectionBinder.bindManyToManySecondPass(CollectionBinder.java:1258)
	at org.hibernate.cfg.annotations.CollectionBinder.bindStarToManySecondPass(CollectionBinder.java:710)
	at org.hibernate.cfg.annotations.CollectionBinder$1.secondPass(CollectionBinder.java:645)
	at org.hibernate.cfg.CollectionSecondPass.doSecondPass(CollectionSecondPass.java:65)
	at org.hibernate.cfg.Configuration.originalSecondPassCompile(Configuration.java:1716)
	at org.hibernate.cfg.Configuration.secondPassCompile(Configuration.java:1423)
	at org.hibernate.cfg.Configuration.buildSessionFactory(Configuration.java:1856)
	at conversorbdweka.hibernate.HibernateUtil.<clinit>(HibernateUtil.java:40)
	at conversorbdweka.dominio.origem.daoimpl.DaoBase.findAll(DaoBase.java:112)
	at conversorbdweka.ConversorBdWeka.main(ConversorBdWeka.java:27)
2015-05-01 22:52:15,797 [main] ERROR conversorbdweka.utilidades.DAOException:27 - 
conversorbdweka.utilidades.DAOException
	at conversorbdweka.dominio.origem.daoimpl.DaoBase.findAll(DaoBase.java:114)
	at conversorbdweka.ConversorBdWeka.main(ConversorBdWeka.java:27)
2015-05-01 22:54:12,688 [main] ERROR conversorbdweka.hibernate.HibernateUtil:42 - Unable to find physical table: `ESPECIALISTA`
org.hibernate.MappingException: Unable to find physical table: `ESPECIALISTA`
	at org.hibernate.cfg.Configuration$MappingsImpl.getLogicalTableName(Configuration.java:3430)
	at org.hibernate.cfg.Configuration$MappingsImpl.getLogicalTableName(Configuration.java:3423)
	at org.hibernate.cfg.annotations.CollectionBinder.bindManyToManySecondPass(CollectionBinder.java:1258)
	at org.hibernate.cfg.annotations.CollectionBinder.bindStarToManySecondPass(CollectionBinder.java:710)
	at org.hibernate.cfg.annotations.CollectionBinder$1.secondPass(CollectionBinder.java:645)
	at org.hibernate.cfg.CollectionSecondPass.doSecondPass(CollectionSecondPass.java:65)
	at org.hibernate.cfg.Configuration.originalSecondPassCompile(Configuration.java:1716)
	at org.hibernate.cfg.Configuration.secondPassCompile(Configuration.java:1423)
	at org.hibernate.cfg.Configuration.buildSessionFactory(Configuration.java:1856)
	at conversorbdweka.hibernate.HibernateUtil.<clinit>(HibernateUtil.java:40)
	at conversorbdweka.dominio.origem.daoimpl.DaoBase.findAll(DaoBase.java:112)
	at conversorbdweka.ConversorBdWeka.main(ConversorBdWeka.java:27)
2015-05-01 22:54:12,723 [main] ERROR conversorbdweka.utilidades.DAOException:27 - 
conversorbdweka.utilidades.DAOException
	at conversorbdweka.dominio.origem.daoimpl.DaoBase.findAll(DaoBase.java:114)
	at conversorbdweka.ConversorBdWeka.main(ConversorBdWeka.java:27)
2015-05-01 22:54:40,336 [main] ERROR conversorbdweka.hibernate.HibernateUtil:42 - Unable to find physical table: `ESPECIALISTA`
org.hibernate.MappingException: Unable to find physical table: `ESPECIALISTA`
	at org.hibernate.cfg.Configuration$MappingsImpl.getLogicalTableName(Configuration.java:3430)
	at org.hibernate.cfg.Configuration$MappingsImpl.getLogicalTableName(Configuration.java:3423)
	at org.hibernate.cfg.annotations.CollectionBinder.bindManyToManySecondPass(CollectionBinder.java:1258)
	at org.hibernate.cfg.annotations.CollectionBinder.bindStarToManySecondPass(CollectionBinder.java:710)
	at org.hibernate.cfg.annotations.CollectionBinder$1.secondPass(CollectionBinder.java:645)
	at org.hibernate.cfg.CollectionSecondPass.doSecondPass(CollectionSecondPass.java:65)
	at org.hibernate.cfg.Configuration.originalSecondPassCompile(Configuration.java:1716)
	at org.hibernate.cfg.Configuration.secondPassCompile(Configuration.java:1423)
	at org.hibernate.cfg.Configuration.buildSessionFactory(Configuration.java:1856)
	at conversorbdweka.hibernate.HibernateUtil.<clinit>(HibernateUtil.java:40)
	at conversorbdweka.dominio.origem.daoimpl.DaoBase.findAll(DaoBase.java:112)
	at conversorbdweka.ConversorBdWeka.main(ConversorBdWeka.java:27)
2015-05-01 22:54:40,361 [main] ERROR conversorbdweka.utilidades.DAOException:27 - 
conversorbdweka.utilidades.DAOException
	at conversorbdweka.dominio.origem.daoimpl.DaoBase.findAll(DaoBase.java:114)
	at conversorbdweka.ConversorBdWeka.main(ConversorBdWeka.java:27)
2015-05-01 22:58:01,705 [main] ERROR conversorbdweka.hibernate.HibernateUtil:42 - Unable to find physical table: `ESPECIALISTA`
org.hibernate.MappingException: Unable to find physical table: `ESPECIALISTA`
	at org.hibernate.cfg.Configuration$MappingsImpl.getLogicalTableName(Configuration.java:3430)
	at org.hibernate.cfg.Configuration$MappingsImpl.getLogicalTableName(Configuration.java:3423)
	at org.hibernate.cfg.annotations.CollectionBinder.bindManyToManySecondPass(CollectionBinder.java:1258)
	at org.hibernate.cfg.annotations.CollectionBinder.bindStarToManySecondPass(CollectionBinder.java:710)
	at org.hibernate.cfg.annotations.CollectionBinder$1.secondPass(CollectionBinder.java:645)
	at org.hibernate.cfg.CollectionSecondPass.doSecondPass(CollectionSecondPass.java:65)
	at org.hibernate.cfg.Configuration.originalSecondPassCompile(Configuration.java:1716)
	at org.hibernate.cfg.Configuration.secondPassCompile(Configuration.java:1423)
	at org.hibernate.cfg.Configuration.buildSessionFactory(Configuration.java:1856)
	at conversorbdweka.hibernate.HibernateUtil.<clinit>(HibernateUtil.java:40)
	at conversorbdweka.dominio.origem.daoimpl.DaoBase.findAll(DaoBase.java:112)
	at conversorbdweka.ConversorBdWeka.main(ConversorBdWeka.java:27)
2015-05-01 22:59:13,666 [main] ERROR conversorbdweka.hibernate.HibernateUtil:42 - Unable to find physical table: `ESPECIALISTA`
org.hibernate.MappingException: Unable to find physical table: `ESPECIALISTA`
	at org.hibernate.cfg.Configuration$MappingsImpl.getLogicalTableName(Configuration.java:3430)
	at org.hibernate.cfg.Configuration$MappingsImpl.getLogicalTableName(Configuration.java:3423)
	at org.hibernate.cfg.annotations.CollectionBinder.bindManyToManySecondPass(CollectionBinder.java:1258)
	at org.hibernate.cfg.annotations.CollectionBinder.bindStarToManySecondPass(CollectionBinder.java:710)
	at org.hibernate.cfg.annotations.CollectionBinder$1.secondPass(CollectionBinder.java:645)
	at org.hibernate.cfg.CollectionSecondPass.doSecondPass(CollectionSecondPass.java:65)
	at org.hibernate.cfg.Configuration.originalSecondPassCompile(Configuration.java:1716)
	at org.hibernate.cfg.Configuration.secondPassCompile(Configuration.java:1423)
	at org.hibernate.cfg.Configuration.buildSessionFactory(Configuration.java:1856)
	at conversorbdweka.hibernate.HibernateUtil.<clinit>(HibernateUtil.java:40)
	at conversorbdweka.dominio.origem.daoimpl.DaoBase.findAll(DaoBase.java:112)
	at conversorbdweka.ConversorBdWeka.main(ConversorBdWeka.java:27)
2015-05-01 23:02:33,463 [main] ERROR conversorbdweka.hibernate.HibernateUtil:42 - Unable to find physical table: `ESPECIALISTA`
org.hibernate.MappingException: Unable to find physical table: `ESPECIALISTA`
	at org.hibernate.cfg.Configuration$MappingsImpl.getLogicalTableName(Configuration.java:3430)
	at org.hibernate.cfg.Configuration$MappingsImpl.getLogicalTableName(Configuration.java:3423)
	at org.hibernate.cfg.annotations.CollectionBinder.bindManyToManySecondPass(CollectionBinder.java:1258)
	at org.hibernate.cfg.annotations.CollectionBinder.bindStarToManySecondPass(CollectionBinder.java:710)
	at org.hibernate.cfg.annotations.CollectionBinder$1.secondPass(CollectionBinder.java:645)
	at org.hibernate.cfg.CollectionSecondPass.doSecondPass(CollectionSecondPass.java:65)
	at org.hibernate.cfg.Configuration.originalSecondPassCompile(Configuration.java:1716)
	at org.hibernate.cfg.Configuration.secondPassCompile(Configuration.java:1423)
	at org.hibernate.cfg.Configuration.buildSessionFactory(Configuration.java:1856)
	at conversorbdweka.hibernate.HibernateUtil.<clinit>(HibernateUtil.java:40)
	at conversorbdweka.dominio.origem.daoimpl.DaoBase.findAll(DaoBase.java:112)
	at conversorbdweka.ConversorBdWeka.main(ConversorBdWeka.java:27)
2015-05-01 23:02:34,787 [main] ERROR conversorbdweka.hibernate.HibernateUtil:42 - Unable to find physical table: `ESPECIALISTA`
org.hibernate.MappingException: Unable to find physical table: `ESPECIALISTA`
	at org.hibernate.cfg.Configuration$MappingsImpl.getLogicalTableName(Configuration.java:3430)
	at org.hibernate.cfg.Configuration$MappingsImpl.getLogicalTableName(Configuration.java:3423)
	at org.hibernate.cfg.annotations.CollectionBinder.bindManyToManySecondPass(CollectionBinder.java:1258)
	at org.hibernate.cfg.annotations.CollectionBinder.bindStarToManySecondPass(CollectionBinder.java:710)
	at org.hibernate.cfg.annotations.CollectionBinder$1.secondPass(CollectionBinder.java:645)
	at org.hibernate.cfg.CollectionSecondPass.doSecondPass(CollectionSecondPass.java:65)
	at org.hibernate.cfg.Configuration.originalSecondPassCompile(Configuration.java:1716)
	at org.hibernate.cfg.Configuration.secondPassCompile(Configuration.java:1423)
	at org.hibernate.cfg.Configuration.buildSessionFactory(Configuration.java:1856)
	at conversorbdweka.hibernate.HibernateUtil.<clinit>(HibernateUtil.java:40)
	at conversorbdweka.dominio.origem.daoimpl.DaoBase.findAll(DaoBase.java:112)
	at conversorbdweka.ConversorBdWeka.main(ConversorBdWeka.java:27)
2015-05-01 23:02:53,088 [main] ERROR conversorbdweka.hibernate.HibernateUtil:42 - Unable to find physical table: `ESPECIALISTA`
org.hibernate.MappingException: Unable to find physical table: `ESPECIALISTA`
	at org.hibernate.cfg.Configuration$MappingsImpl.getLogicalTableName(Configuration.java:3430)
	at org.hibernate.cfg.Configuration$MappingsImpl.getLogicalTableName(Configuration.java:3423)
	at org.hibernate.cfg.annotations.CollectionBinder.bindManyToManySecondPass(CollectionBinder.java:1258)
	at org.hibernate.cfg.annotations.CollectionBinder.bindStarToManySecondPass(CollectionBinder.java:710)
	at org.hibernate.cfg.annotations.CollectionBinder$1.secondPass(CollectionBinder.java:645)
	at org.hibernate.cfg.CollectionSecondPass.doSecondPass(CollectionSecondPass.java:65)
	at org.hibernate.cfg.Configuration.originalSecondPassCompile(Configuration.java:1716)
	at org.hibernate.cfg.Configuration.secondPassCompile(Configuration.java:1423)
	at org.hibernate.cfg.Configuration.buildSessionFactory(Configuration.java:1856)
	at conversorbdweka.hibernate.HibernateUtil.<clinit>(HibernateUtil.java:40)
	at conversorbdweka.dominio.origem.daoimpl.DaoBase.findAll(DaoBase.java:112)
	at conversorbdweka.ConversorBdWeka.main(ConversorBdWeka.java:27)
2015-05-01 23:06:40,911 [main] ERROR conversorbdweka.hibernate.HibernateUtil:42 - Unable to find physical table: `ESPECIALISTA`
org.hibernate.MappingException: Unable to find physical table: `ESPECIALISTA`
	at org.hibernate.cfg.Configuration$MappingsImpl.getLogicalTableName(Configuration.java:3430)
	at org.hibernate.cfg.Configuration$MappingsImpl.getLogicalTableName(Configuration.java:3423)
	at org.hibernate.cfg.annotations.CollectionBinder.bindManyToManySecondPass(CollectionBinder.java:1258)
	at org.hibernate.cfg.annotations.CollectionBinder.bindStarToManySecondPass(CollectionBinder.java:710)
	at org.hibernate.cfg.annotations.CollectionBinder$1.secondPass(CollectionBinder.java:645)
	at org.hibernate.cfg.CollectionSecondPass.doSecondPass(CollectionSecondPass.java:65)
	at org.hibernate.cfg.Configuration.originalSecondPassCompile(Configuration.java:1716)
	at org.hibernate.cfg.Configuration.secondPassCompile(Configuration.java:1423)
	at org.hibernate.cfg.Configuration.buildSessionFactory(Configuration.java:1856)
	at conversorbdweka.hibernate.HibernateUtil.<clinit>(HibernateUtil.java:40)
	at conversorbdweka.dominio.origem.daoimpl.DaoBase.findAll(DaoBase.java:112)
	at conversorbdweka.ConversorBdWeka.main(ConversorBdWeka.java:27)
2015-05-01 23:06:49,751 [main] ERROR conversorbdweka.hibernate.HibernateUtil:42 - Unable to find physical table: `ESPECIALISTA`
org.hibernate.MappingException: Unable to find physical table: `ESPECIALISTA`
	at org.hibernate.cfg.Configuration$MappingsImpl.getLogicalTableName(Configuration.java:3430)
	at org.hibernate.cfg.Configuration$MappingsImpl.getLogicalTableName(Configuration.java:3423)
	at org.hibernate.cfg.annotations.CollectionBinder.bindManyToManySecondPass(CollectionBinder.java:1258)
	at org.hibernate.cfg.annotations.CollectionBinder.bindStarToManySecondPass(CollectionBinder.java:710)
	at org.hibernate.cfg.annotations.CollectionBinder$1.secondPass(CollectionBinder.java:645)
	at org.hibernate.cfg.CollectionSecondPass.doSecondPass(CollectionSecondPass.java:65)
	at org.hibernate.cfg.Configuration.originalSecondPassCompile(Configuration.java:1716)
	at org.hibernate.cfg.Configuration.secondPassCompile(Configuration.java:1423)
	at org.hibernate.cfg.Configuration.buildSessionFactory(Configuration.java:1856)
	at conversorbdweka.hibernate.HibernateUtil.<clinit>(HibernateUtil.java:40)
	at conversorbdweka.dominio.origem.daoimpl.DaoBase.findAll(DaoBase.java:112)
	at conversorbdweka.ConversorBdWeka.main(ConversorBdWeka.java:27)
2015-05-01 23:08:07,276 [main] ERROR conversorbdweka.hibernate.HibernateUtil:42 - Unable to find physical table: `ESPECIALISTA`
org.hibernate.MappingException: Unable to find physical table: `ESPECIALISTA`
	at org.hibernate.cfg.Configuration$MappingsImpl.getLogicalTableName(Configuration.java:3430)
	at org.hibernate.cfg.Configuration$MappingsImpl.getLogicalTableName(Configuration.java:3423)
	at org.hibernate.cfg.annotations.CollectionBinder.bindManyToManySecondPass(CollectionBinder.java:1258)
	at org.hibernate.cfg.annotations.CollectionBinder.bindStarToManySecondPass(CollectionBinder.java:710)
	at org.hibernate.cfg.annotations.CollectionBinder$1.secondPass(CollectionBinder.java:645)
	at org.hibernate.cfg.CollectionSecondPass.doSecondPass(CollectionSecondPass.java:65)
	at org.hibernate.cfg.Configuration.originalSecondPassCompile(Configuration.java:1716)
	at org.hibernate.cfg.Configuration.secondPassCompile(Configuration.java:1423)
	at org.hibernate.cfg.Configuration.buildSessionFactory(Configuration.java:1856)
	at conversorbdweka.hibernate.HibernateUtil.<clinit>(HibernateUtil.java:40)
	at conversorbdweka.dominio.origem.daoimpl.DaoBase.findAll(DaoBase.java:112)
	at conversorbdweka.ConversorBdWeka.main(ConversorBdWeka.java:27)
2015-05-01 23:08:27,585 [main] ERROR conversorbdweka.hibernate.HibernateUtil:42 - Unable to find physical table: `ESPECIALISTAa`
org.hibernate.MappingException: Unable to find physical table: `ESPECIALISTAa`
	at org.hibernate.cfg.Configuration$MappingsImpl.getLogicalTableName(Configuration.java:3430)
	at org.hibernate.cfg.Configuration$MappingsImpl.getLogicalTableName(Configuration.java:3423)
	at org.hibernate.cfg.annotations.CollectionBinder.bindManyToManySecondPass(CollectionBinder.java:1258)
	at org.hibernate.cfg.annotations.CollectionBinder.bindStarToManySecondPass(CollectionBinder.java:710)
	at org.hibernate.cfg.annotations.CollectionBinder$1.secondPass(CollectionBinder.java:645)
	at org.hibernate.cfg.CollectionSecondPass.doSecondPass(CollectionSecondPass.java:65)
	at org.hibernate.cfg.Configuration.originalSecondPassCompile(Configuration.java:1716)
	at org.hibernate.cfg.Configuration.secondPassCompile(Configuration.java:1423)
	at org.hibernate.cfg.Configuration.buildSessionFactory(Configuration.java:1856)
	at conversorbdweka.hibernate.HibernateUtil.<clinit>(HibernateUtil.java:40)
	at conversorbdweka.dominio.origem.daoimpl.DaoBase.findAll(DaoBase.java:112)
	at conversorbdweka.ConversorBdWeka.main(ConversorBdWeka.java:27)
2015-05-01 23:14:51,535 [main] ERROR conversorbdweka.hibernate.HibernateUtil:42 - Unable to find physical table: ESPECIALISTA
org.hibernate.MappingException: Unable to find physical table: ESPECIALISTA
	at org.hibernate.cfg.Configuration$MappingsImpl.getLogicalTableName(Configuration.java:3430)
	at org.hibernate.cfg.Configuration$MappingsImpl.getLogicalTableName(Configuration.java:3423)
	at org.hibernate.cfg.annotations.CollectionBinder.bindManyToManySecondPass(CollectionBinder.java:1258)
	at org.hibernate.cfg.annotations.CollectionBinder.bindStarToManySecondPass(CollectionBinder.java:710)
	at org.hibernate.cfg.annotations.CollectionBinder$1.secondPass(CollectionBinder.java:645)
	at org.hibernate.cfg.CollectionSecondPass.doSecondPass(CollectionSecondPass.java:65)
	at org.hibernate.cfg.Configuration.originalSecondPassCompile(Configuration.java:1716)
	at org.hibernate.cfg.Configuration.secondPassCompile(Configuration.java:1423)
	at org.hibernate.cfg.Configuration.buildSessionFactory(Configuration.java:1856)
	at conversorbdweka.hibernate.HibernateUtil.<clinit>(HibernateUtil.java:40)
	at conversorbdweka.dominio.origem.daoimpl.DaoBase.findAll(DaoBase.java:112)
	at conversorbdweka.ConversorBdWeka.main(ConversorBdWeka.java:27)
2015-05-01 23:19:20,201 [main] ERROR conversorbdweka.hibernate.HibernateUtil:42 - Unable to find physical table: ESPECIALISTA
org.hibernate.MappingException: Unable to find physical table: ESPECIALISTA
	at org.hibernate.cfg.Configuration$MappingsImpl.getLogicalTableName(Configuration.java:3430)
	at org.hibernate.cfg.Configuration$MappingsImpl.getLogicalTableName(Configuration.java:3423)
	at org.hibernate.cfg.annotations.CollectionBinder.bindManyToManySecondPass(CollectionBinder.java:1258)
	at org.hibernate.cfg.annotations.CollectionBinder.bindStarToManySecondPass(CollectionBinder.java:710)
	at org.hibernate.cfg.annotations.CollectionBinder$1.secondPass(CollectionBinder.java:645)
	at org.hibernate.cfg.CollectionSecondPass.doSecondPass(CollectionSecondPass.java:65)
	at org.hibernate.cfg.Configuration.originalSecondPassCompile(Configuration.java:1716)
	at org.hibernate.cfg.Configuration.secondPassCompile(Configuration.java:1423)
	at org.hibernate.cfg.Configuration.buildSessionFactory(Configuration.java:1856)
	at conversorbdweka.hibernate.HibernateUtil.<clinit>(HibernateUtil.java:40)
	at conversorbdweka.dominio.origem.daoimpl.DaoBase.findAll(DaoBase.java:112)
	at conversorbdweka.ConversorBdWeka.main(ConversorBdWeka.java:27)
2015-05-01 23:23:02,898 [main] ERROR conversorbdweka.hibernate.HibernateUtil:42 - Unable to find physical table: ESPECIALISTA
org.hibernate.MappingException: Unable to find physical table: ESPECIALISTA
	at org.hibernate.cfg.Configuration$MappingsImpl.getLogicalTableName(Configuration.java:3430)
	at org.hibernate.cfg.Configuration$MappingsImpl.getLogicalTableName(Configuration.java:3423)
	at org.hibernate.cfg.annotations.CollectionBinder.bindManyToManySecondPass(CollectionBinder.java:1258)
	at org.hibernate.cfg.annotations.CollectionBinder.bindStarToManySecondPass(CollectionBinder.java:710)
	at org.hibernate.cfg.annotations.CollectionBinder$1.secondPass(CollectionBinder.java:645)
	at org.hibernate.cfg.CollectionSecondPass.doSecondPass(CollectionSecondPass.java:65)
	at org.hibernate.cfg.Configuration.originalSecondPassCompile(Configuration.java:1716)
	at org.hibernate.cfg.Configuration.secondPassCompile(Configuration.java:1423)
	at org.hibernate.cfg.Configuration.buildSessionFactory(Configuration.java:1856)
	at conversorbdweka.hibernate.HibernateUtil.<clinit>(HibernateUtil.java:40)
	at conversorbdweka.dominio.origem.daoimpl.DaoBase.findAll(DaoBase.java:112)
	at conversorbdweka.ConversorBdWeka.main(ConversorBdWeka.java:29)
2015-05-01 23:26:04,384 [main] ERROR conversorbdweka.hibernate.HibernateUtil:42 - Unable to find physical table: ESPECIALISTA
org.hibernate.MappingException: Unable to find physical table: ESPECIALISTA
	at org.hibernate.cfg.Configuration$MappingsImpl.getLogicalTableName(Configuration.java:3430)
	at org.hibernate.cfg.Configuration$MappingsImpl.getLogicalTableName(Configuration.java:3423)
	at org.hibernate.cfg.annotations.CollectionBinder.bindManyToManySecondPass(CollectionBinder.java:1258)
	at org.hibernate.cfg.annotations.CollectionBinder.bindStarToManySecondPass(CollectionBinder.java:710)
	at org.hibernate.cfg.annotations.CollectionBinder$1.secondPass(CollectionBinder.java:645)
	at org.hibernate.cfg.CollectionSecondPass.doSecondPass(CollectionSecondPass.java:65)
	at org.hibernate.cfg.Configuration.originalSecondPassCompile(Configuration.java:1716)
	at org.hibernate.cfg.Configuration.secondPassCompile(Configuration.java:1423)
	at org.hibernate.cfg.Configuration.buildSessionFactory(Configuration.java:1856)
	at conversorbdweka.hibernate.HibernateUtil.<clinit>(HibernateUtil.java:40)
	at conversorbdweka.dominio.origem.daoimpl.DaoBase.findAll(DaoBase.java:112)
	at conversorbdweka.ConversorBdWeka.main(ConversorBdWeka.java:30)
2015-05-01 23:29:46,590 [main] ERROR conversorbdweka.utilidades.DAOException:27 - could not execute query
conversorbdweka.utilidades.DAOException: could not execute query
	at conversorbdweka.dominio.origem.daoimpl.DaoBase.findAll(DaoBase.java:114)
	at conversorbdweka.ConversorBdWeka.main(ConversorBdWeka.java:30)
2015-05-01 23:32:06,391 [main] ERROR conversorbdweka.utilidades.DAOException:27 - could not execute query
conversorbdweka.utilidades.DAOException: could not execute query
	at conversorbdweka.dominio.origem.daoimpl.DaoBase.findAll(DaoBase.java:114)
	at conversorbdweka.ConversorBdWeka.main(ConversorBdWeka.java:30)
2015-05-01 23:35:02,913 [main] ERROR conversorbdweka.hibernate.HibernateUtil:42 - Unable to find physical table: ESPECIALISTA
org.hibernate.MappingException: Unable to find physical table: ESPECIALISTA
	at org.hibernate.cfg.Configuration$MappingsImpl.getLogicalTableName(Configuration.java:3430)
	at org.hibernate.cfg.Configuration$MappingsImpl.getLogicalTableName(Configuration.java:3423)
	at org.hibernate.cfg.annotations.CollectionBinder.bindManyToManySecondPass(CollectionBinder.java:1258)
	at org.hibernate.cfg.annotations.CollectionBinder.bindStarToManySecondPass(CollectionBinder.java:710)
	at org.hibernate.cfg.annotations.CollectionBinder$1.secondPass(CollectionBinder.java:645)
	at org.hibernate.cfg.CollectionSecondPass.doSecondPass(CollectionSecondPass.java:65)
	at org.hibernate.cfg.Configuration.originalSecondPassCompile(Configuration.java:1716)
	at org.hibernate.cfg.Configuration.secondPassCompile(Configuration.java:1423)
	at org.hibernate.cfg.Configuration.buildSessionFactory(Configuration.java:1856)
	at conversorbdweka.hibernate.HibernateUtil.<clinit>(HibernateUtil.java:40)
	at conversorbdweka.dominio.origem.daoimpl.DaoBase.findAll(DaoBase.java:112)
	at conversorbdweka.ConversorBdWeka.main(ConversorBdWeka.java:30)
2015-05-01 23:35:49,621 [main] ERROR conversorbdweka.hibernate.HibernateUtil:42 - Unable to find physical table: `FRASE_ESPECIALISTA_ESPECIALIDADE`
org.hibernate.MappingException: Unable to find physical table: `FRASE_ESPECIALISTA_ESPECIALIDADE`
	at org.hibernate.cfg.Configuration$MappingsImpl.getLogicalTableName(Configuration.java:3430)
	at org.hibernate.cfg.Configuration$MappingsImpl.getLogicalTableName(Configuration.java:3423)
	at org.hibernate.cfg.annotations.CollectionBinder.bindManyToManySecondPass(CollectionBinder.java:1267)
	at org.hibernate.cfg.annotations.CollectionBinder.bindStarToManySecondPass(CollectionBinder.java:710)
	at org.hibernate.cfg.annotations.CollectionBinder$1.secondPass(CollectionBinder.java:645)
	at org.hibernate.cfg.CollectionSecondPass.doSecondPass(CollectionSecondPass.java:65)
	at org.hibernate.cfg.Configuration.originalSecondPassCompile(Configuration.java:1716)
	at org.hibernate.cfg.Configuration.secondPassCompile(Configuration.java:1423)
	at org.hibernate.cfg.Configuration.buildSessionFactory(Configuration.java:1856)
	at conversorbdweka.hibernate.HibernateUtil.<clinit>(HibernateUtil.java:40)
	at conversorbdweka.dominio.origem.daoimpl.DaoBase.findAll(DaoBase.java:112)
	at conversorbdweka.ConversorBdWeka.main(ConversorBdWeka.java:30)
2015-05-01 23:43:56,845 [main] ERROR conversorbdweka.utilidades.DAOException:27 - could not execute query
conversorbdweka.utilidades.DAOException: could not execute query
	at conversorbdweka.dominio.origem.daoimpl.DaoBase.findAll(DaoBase.java:114)
	at conversorbdweka.ConversorBdWeka.main(ConversorBdWeka.java:30)
2015-05-02 08:21:43,743 [main] ERROR br.ufmg.hc.telessaude.diagnostico.dominio.hibernate.HibernateUtil:42 - Unable to find physical table: `ESPECIALISTA`
org.hibernate.MappingException: Unable to find physical table: `ESPECIALISTA`
	at org.hibernate.cfg.Configuration$MappingsImpl.getLogicalTableName(Configuration.java:3430)
	at org.hibernate.cfg.Configuration$MappingsImpl.getLogicalTableName(Configuration.java:3423)
	at org.hibernate.cfg.annotations.CollectionBinder.bindManyToManySecondPass(CollectionBinder.java:1258)
	at org.hibernate.cfg.annotations.CollectionBinder.bindStarToManySecondPass(CollectionBinder.java:710)
	at org.hibernate.cfg.annotations.CollectionBinder$1.secondPass(CollectionBinder.java:645)
	at org.hibernate.cfg.CollectionSecondPass.doSecondPass(CollectionSecondPass.java:65)
	at org.hibernate.cfg.Configuration.originalSecondPassCompile(Configuration.java:1716)
	at org.hibernate.cfg.Configuration.secondPassCompile(Configuration.java:1423)
	at org.hibernate.cfg.Configuration.buildSessionFactory(Configuration.java:1856)
	at br.ufmg.hc.telessaude.diagnostico.dominio.hibernate.HibernateUtil.<clinit>(HibernateUtil.java:40)
	at br.ufmg.hc.telessaude.diagnostico.dominio.daoImpl.DaoBase.findAll(DaoBase.java:112)
	at conversorbdweka.ConversorBdWeka.main(ConversorBdWeka.java:27)
2015-05-02 08:21:43,771 [main] ERROR br.ufmg.hc.telessaude.diagnostico.dominio.exceptions.DAOException:29 - 
br.ufmg.hc.telessaude.diagnostico.dominio.exceptions.DAOException
	at br.ufmg.hc.telessaude.diagnostico.dominio.daoImpl.DaoBase.findAll(DaoBase.java:114)
	at conversorbdweka.ConversorBdWeka.main(ConversorBdWeka.java:27)
2015-05-02 08:29:20,883 [main] ERROR br.ufmg.hc.telessaude.diagnostico.dominio.hibernate.HibernateUtil:42 - Unable to find physical table: `ESPECIALISTA`
org.hibernate.MappingException: Unable to find physical table: `ESPECIALISTA`
	at org.hibernate.cfg.Configuration$MappingsImpl.getLogicalTableName(Configuration.java:3430)
	at org.hibernate.cfg.Configuration$MappingsImpl.getLogicalTableName(Configuration.java:3423)
	at org.hibernate.cfg.annotations.CollectionBinder.bindManyToManySecondPass(CollectionBinder.java:1258)
	at org.hibernate.cfg.annotations.CollectionBinder.bindStarToManySecondPass(CollectionBinder.java:710)
	at org.hibernate.cfg.annotations.CollectionBinder$1.secondPass(CollectionBinder.java:645)
	at org.hibernate.cfg.CollectionSecondPass.doSecondPass(CollectionSecondPass.java:65)
	at org.hibernate.cfg.Configuration.originalSecondPassCompile(Configuration.java:1716)
	at org.hibernate.cfg.Configuration.secondPassCompile(Configuration.java:1423)
	at org.hibernate.cfg.Configuration.buildSessionFactory(Configuration.java:1856)
	at br.ufmg.hc.telessaude.diagnostico.dominio.hibernate.HibernateUtil.<clinit>(HibernateUtil.java:40)
	at br.ufmg.hc.telessaude.diagnostico.dominio.daoImpl.DaoBase.findAll(DaoBase.java:112)
	at conversorbdweka.ConversorBdWeka.main(ConversorBdWeka.java:27)
2015-05-02 08:29:20,906 [main] ERROR br.ufmg.hc.telessaude.diagnostico.dominio.exceptions.DAOException:29 - 
br.ufmg.hc.telessaude.diagnostico.dominio.exceptions.DAOException
	at br.ufmg.hc.telessaude.diagnostico.dominio.daoImpl.DaoBase.findAll(DaoBase.java:114)
	at conversorbdweka.ConversorBdWeka.main(ConversorBdWeka.java:27)
2015-05-02 08:30:34,050 [main] ERROR br.ufmg.hc.telessaude.diagnostico.dominio.hibernate.HibernateUtil:42 - Unable to find physical table: `ESPECIALISTA`
org.hibernate.MappingException: Unable to find physical table: `ESPECIALISTA`
	at org.hibernate.cfg.Configuration$MappingsImpl.getLogicalTableName(Configuration.java:3430)
	at org.hibernate.cfg.Configuration$MappingsImpl.getLogicalTableName(Configuration.java:3423)
	at org.hibernate.cfg.annotations.CollectionBinder.bindManyToManySecondPass(CollectionBinder.java:1258)
	at org.hibernate.cfg.annotations.CollectionBinder.bindStarToManySecondPass(CollectionBinder.java:710)
	at org.hibernate.cfg.annotations.CollectionBinder$1.secondPass(CollectionBinder.java:645)
	at org.hibernate.cfg.CollectionSecondPass.doSecondPass(CollectionSecondPass.java:65)
	at org.hibernate.cfg.Configuration.originalSecondPassCompile(Configuration.java:1716)
	at org.hibernate.cfg.Configuration.secondPassCompile(Configuration.java:1423)
	at org.hibernate.cfg.Configuration.buildSessionFactory(Configuration.java:1856)
	at br.ufmg.hc.telessaude.diagnostico.dominio.hibernate.HibernateUtil.<clinit>(HibernateUtil.java:40)
	at br.ufmg.hc.telessaude.diagnostico.dominio.daoImpl.DaoBase.findAll(DaoBase.java:112)
	at conversorbdweka.ConversorBdWeka.main(ConversorBdWeka.java:27)
2015-05-02 08:30:34,094 [main] ERROR br.ufmg.hc.telessaude.diagnostico.dominio.exceptions.DAOException:29 - 
br.ufmg.hc.telessaude.diagnostico.dominio.exceptions.DAOException
	at br.ufmg.hc.telessaude.diagnostico.dominio.daoImpl.DaoBase.findAll(DaoBase.java:114)
	at conversorbdweka.ConversorBdWeka.main(ConversorBdWeka.java:27)
